eng/pipelines/libraries/helix-queues-setup.yml

```diff
@@ -77,7 +77,7 @@ jobs:
             - Ubuntu.1804.Amd64.Open
             - SLES.15.Amd64.Open
             - (Fedora.30.Amd64.Open)ubuntu.1604.amd64.open@mcr.microsoft.com/dotnet-buildtools/prereqs:fedora-30-helix-20200512010621-4f8cef7
-      - ${{ if eq(parameters.jobParameters.interpreter, 'true') }}:
+      - ${{ if or(eq(parameters.jobParameters.isSingleFile, true), eq(parameters.jobParameters.interpreter, 'true') }}:
```

Comment:
You'll need to expand the condition above as well so that all the previous queues don't get included

Reply:
Also, I think we should just pick one windows queue for this (just to not add more extra load to our windows queues.